### PR TITLE
Create new "aktualizr-get" command

### PR DESCRIPTION
**Pull request by doanac**
I find myself carefully crafting `wget` commands to debug issues
between my device and device-gateway. This has gotten more complex
now that my team is using HSMs. This adds a simple tool to work
a bit like wget.

Signed-off-by: Andy Doan <andy@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Feel free to reject this. Its easy enough for my team to manage out-of-tree. However, I thought you guys might find it useful also.

File: src/aktualizr_get/main.cc [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+#include "config/config.h"
+#include "get.h"
+
+#include "utilities/aktualizr_version.h"
+
+namespace bpo = boost::program_options;
+
+bpo::variables_map parse_options(int argc, char *argv[]) {
+  bpo::options_description description("aktualizr-get command line options");
```
Comment (patrickvacek) [inline src/aktualizr_get/main.cc]: Might be useful to either print something here or at least have a comment to explain what the tool is for.

File: src/aktualizr_get/main.cc [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+int main(int argc, char *argv[]) {
+  logger_init(isatty(1) == 1);
+  logger_set_threshold(boost::log::trivial::info);
+
+  bpo::variables_map commandline_map = parse_options(argc, argv);
+
+  int r = EXIT_FAILURE;
+  try {
+    if (geteuid() != 0) {
+      LOG_WARNING << "\033[31mRunning as non-root and may not work as expected!\033[0m\n";
```
Comment (patrickvacek) [inline src/aktualizr_get/main.cc]: Is that true? I don't think it applies to this tool.

**Comment by doanac** [comment]
@patrickvacek - good points. fixed now.

**Comment by patrickvacek** [comment]
Looks good, but CI is still complaining about a compilation error.

**Comment by patrickvacek** [comment]
Looks like you fixed the compilation error, but accidentally changed the version of tuf-test-vectors, so now that is failing. If you undo that bit, it'll probably succeed.

**Comment by doanac** [comment]
`git commit -a --amend` can be evil. sorry about that.

**Comment by patrickvacek** [comment]
Travis is timing out, which is annoying, but I tested it locally and it's fine.

**Pull request merged**